build-system-tests/templates/components/react/next/template-package-11.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "react-17-next-11-ts",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@aws-amplify/ui-react": "^4.5.1",
```

Comment:
Should we use `latest` instead of `^4.5.1`?

Reply:
oh, right, they actually should be in the package.json. I'll remove them

Reply:
Done 68132342f3c9922ddb252c75e8d43a39811a38a8